Mathlib/CategoryTheory/Limits/Shapes/Images.lean

```diff
@@ -73,58 +73,59 @@ variable {X Y : C} (f : X ⟶ Y)
 
 /-- A factorisation of a morphism `f = e ≫ m`, with `m` monic. -/
 structure MonoFactorisation (f : X ⟶ Y) where
-  i : C
+  /-- The "image" of f -/
+  I : C -- Porting note: violates naming conventions but can't think a better replacement
+  /-- The inclusion of of the image -/
   m : I ⟶ Y
+  /-- `m` is a monomorphism -/
   [m_mono : Mono m]
+  /-- The map to the image -/
   e : X ⟶ I
-  fac : e ≫ m = f := by obviously
+  /-- The factorisation -/
```

Comment:
There is no need to add docs while porting. So let's not do it for sake of atomic actions :wink: :grinning:

Reply:
`nolint docBlame`?

Reply:
Added `@[inherit_doc]` to all the fields